### PR TITLE
Fix T1034658 - editCellTemplate must contain a DevExtreme editor

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by vconst — APPROVED** [review]

**Review by ilyapavlovdx — APPROVED** [review]

File: api-reference/_hidden/dxDataGridColumn/editCellTemplate.md [hunk 1]
```diff
@@ -69,18 +69,26 @@ Use the **editCellTemplate** to replace the default editor. In the template, spe
 
 Other properties that allow editor customization include:
 
-- **columns[]**.[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions')         
+- **columns[]**.[editorOptions]({basewidgetpath}/Configuration/columns/#editorOptions)         
 Use it to specify the default editor's properties (except **onValueChanged**).
```
Comment (arminal) [inline api-reference/_hidden/dxDataGridColumn/editCellTemplate.md]:
```suggestion
Use it to specify default editor properties (except **onValueChanged**).
```

**Comment by arminal** [comment]
#GrammarReview

Short doc, but I do want you to take a look at how and when to use the possessive. More often than not, you can construct a sentence without using an apostrophe.

File: api-reference/_hidden/dxDataGridColumn/editCellTemplate.md (hunk 1 above)
Comment (RomanTsukanov) [reply api-reference/_hidden/dxDataGridColumn/editCellTemplate.md]: I believe this edit causes ambiguity. Is it "the properties of the default editor" or "the default properties of an editor"?

Comment (arminal) [reply api-reference/_hidden/dxDataGridColumn/editCellTemplate.md]: As you wrote the sentence, I understood it to mean that the properties belong to the default editor. My suggested edit says the same thing.

**Pull request merged**